CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Main (unreleased)
 ### Breaking changes to non-GA functionality
 
 - Update Public preview `remotecfg` argument from `metadata` to `attributes`. (@erikbaranowski)
+- The `beyla.ebpf.routes.unmatched` configuration option default changed from unset to `heuristic` (@marctc) 
```

Comment:
I think that it would also be nice to have in Enhancements an entry that says that Beyla was updated to version 1.7 and list some of the key changes